src/metiq.py

```diff
@@ -56,6 +56,8 @@
     "func": "help",
     "infile": None,
     "outfile": None,
+    "audio_offset": 0,
+    "lock_layout": False,
```

Comment:
can you please add in the patch description what both `audio_offset` and `lock_layout` exactly do?

Reply:
OK.